src/util/unicode.h

```diff
@@ -20,7 +20,6 @@ std::string narrow(const std::wstring &s);
 std::wstring widen(const std::string &s);
 
 std::string utf32_to_utf8(const std::basic_string<unsigned int> &s);
```

Comment:
I don't follow this comment.

Are you confusing this function, which still exist, with a function that got renamed to `utf8_append_code`?